### ISSUE TITLE
Disabling "Rewrite Titles" setting no longer works

### ISSUE THREAD

**Issue by arnaudbroes** [semperfiwebdesign/all-in-one-seo-pack#1391]
The plugin still rewrites titles when the 'Rewrite Titles' setting has been disabled.
I've noticed and confirmed this on an existing site and a fresh install.

**Comment by wpsmort** [comment]
Reported again here - https://semperplugins.com/support/troubleshooting-all-in-one-seo-pack-pro/please-fix-rewrite-titles/#p7880#spEditFormAnchor

**Comment by wpsmort** [comment]
@arnaudbroes Can you see when this bug first occurred, which version of AIOSEOP.

**Comment by michaeltorbert** [comment]
Reported again here: https://twitter.com/hpagents/status/1096050816245481472

**Comment by arnaudbroes** [comment]
Fortunately, this only affects the preview snippet and not the source code.

**Comment by michaeltorbert** [comment]
@arnaudbroes That doesn't seem to be the case here with hpagents 
view-source:https://hpagents.com/about-us-hpagents-com/

**Comment by arnaudbroes** [comment]
This will be no longer relevant after #69 is done.
